backend-test/game/game.js

```diff
@@ -1,7 +1,13 @@
-/* eslint no-undef: 0 */
+/* eslint no-undef: 0, no-unused-vars: 0, no-shadow: 0 */
```

Comment:
Remove this line completely and fix the eslint issues that might result in this file

Reply:
Done